.github/workflows/ci.yml

```diff
@@ -28,25 +28,30 @@ jobs:
           check-path: ${{ matrix.path['check'] }}
           exclude-regex: ${{ matrix.path['exclude'] }}
 
-  cpp-linter:
+  linter:
+    name: Build And Run linter
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: cpp-linter/cpp-linter-action@v2
-        id: linter
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
         with:
-          style: file
-          extensions: 'cpp,h'
-          version: 13
-          ignore: "endToEndTests|build|include/silo/roaring.h"
-          files-changed-only: false
-          tidy-checks: ''
-
-      - name: Fail fast!
-        if: steps.linter.outputs.checks-failed > 0
-        run: echo "Some files failed the linting checks!"
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build linter image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          push: false
+          load: true
```

Comment:
I guess we don't need to load the image, do we? Basically we don't need the image at all, we just want to build inside it?